roboteam_ai/src/control/ControlUtils.cpp

```diff
@@ -232,6 +231,25 @@ Vector2 ControlUtils::velocityLimiter(Vector2 vel, double maxVel,double minVel){
 }
 
 
+/// Limits acceleration to maximum acceleration
+Vector2 ControlUtils::accelerationLimiter(Vector2 vel, double maxAcc,double prevVel){
+    if (vel.length() > (prevVel + maxAcc/Constants::TICK_RATE())) {
```

Comment:
we should probably prevent writing division by tickRate since we can measure the time intervals now between the last worldstates

Reply:
so you could just do maxAcc*time interval